templates/sick/templates.yaml

```diff
@@ -0,0 +1,55 @@
+dataset: sick
+templates:
+  2b5fcfdc-8dc4-4aed-9819-8a104230d0fa: !Template
+    id: 2b5fcfdc-8dc4-4aed-9819-8a104230d0fa
+    jinja: 'How related are the following sentences?
+
+      Give a score on a scale of 1 to 5.
+
+
+      {{sentence_A}}
+
+
+      {{sentence_B}} |||
+
+
+      {{relatedness_score}}
+
+
+      '
+    name: related
+    reference: ''
+  566db154-818a-43c6-b66d-924a20fbbec2: !Template
+    id: 566db154-818a-43c6-b66d-924a20fbbec2
+    jinja: "Does sentence B entail or contradict sentence A?\n\n{{sentence_A}}\n\n\
+      {{sentence_B}} |||\n{{\n[\n  \"entailment\",\n  \"neutral\",\n  \"contradiction\"\
+      \n][label]\n}}"
+    name: entailed
+    reference: ''
+  a502cdc1-3bf0-4019-8b4c-b293d75a95ff: !Template
+    id: a502cdc1-3bf0-4019-8b4c-b293d75a95ff
+    jinja: "Does sentence B, \"{{sentence_B}}\", support sentence A, \"{{sentence_A}}\"\
+      \ ? |||\n{{\n[\n  \"yes, B supports A\",\n  \"neutral\",\n  \"no, B contradicts\
+      \ A\"\n][label]\n}}"
+    name: support
+    reference: ''
+  c65e4a05-3252-4f3b-a10f-4fedeb9a61bb: !Template
+    id: c65e4a05-3252-4f3b-a10f-4fedeb9a61bb
+    jinja: "Does the hypothesis \"{{sentence_B}}\" follow  the premise \"{{sentence_A}}\"\
+      ? |||\n{{\n[\n  \"yes, B entails A\",\n  \"neutral\",\n  \"no, B contradicts\
+      \ A\"\n][label]\n}}"
+    name: hypothesis
+    reference: ''
+  eb368955-dc71-45cb-95fe-8d3fee0da819: !Template
+    id: eb368955-dc71-45cb-95fe-8d3fee0da819
+    jinja: 'What is the entailment of sentences A and B?
+
+
+      {{sentence_A}}
+
+
+      {{sentence_B}} |||
+
+      entailment_AB is {{entailment_AB }} while entailment_BA is {{entailment_BA }}'
```

Comment:
`entailment_AB` is not a natural phrase a human would understand. Can you make it more clear?